src/EnergyPlus/Psychrometrics.hh

```diff
@@ -845,10 +844,10 @@ namespace Psychrometrics {
         // values from PsyRhFnTdbWPb
 
         // FUNCTION PARAMETER DEFINITIONS:
-        static std::string const RoutineName("PsyRhFnTdbRhov");
+        static constexpr std::string_view RoutineName(PsyRoutineNames[static_cast<int>(PsychrometricFunction::RhFnTdbRhov)]); // PsyRhFnTdbRhov
```

Comment:
Keep the `static` keyword? I left `RoutineName`s as separate variables for readability.